### ISSUE TITLE
Neurocomputing special issue on early fidelities

### ISSUE THREAD

**Issue by Deathn0t** [fmohr/lcdb#16]
**Technical TODOs**

- [ ] pick classification datasets on which to run learning curves
- [ ] collect the number of trainable parameters as a metadata for densenn workflow
- [ ] create run functions to benchmark multi-fidelity strategies with lcdb 1.0
- [ ] show from characteristics (scores + clusters) which policy should whin (constant fidelity, SHA, LCE) and therefore which datasets/curves should be used to benchmark X-class of multi-fidelity algorithm
- [ ] present all learning curves with minimization normalization (BCE: balanced error rate for classification, MSE: mean squared error for regression)
- [x] set number of trees in random forest as constant for learning curves of the same length @fmohr 
- [x] set the number of epochs in densenn as constant for learning curves of the same length @Deathn0t 
- [x] remove white color from colormap (can't see the curves otherwise)
- [x] develop the "random"-predictor for classification @fmohr 
- [x] develop the "constant"-predictor for classification (mode of targets, ie always predict the most frequent class in training data) @Deathn0t 
- [x] develop the "constant"-predictor for regression (mean of targets for squared L2 loss, median for absolute L1 loss) @fmohr 
- [x] add constant and random predictor performance for regression/classification learning curves and use it to define the middle of the ranking (colormap)
- [x] also include regression in lcdb 2.0 @fmohr 
- [x] add OOB curve for random forests @fmohr 
- [ ] normalize curves with respect to the constant (classification) or constant (regression) predictor (aka, the "simplest" thing to do), this predictor should also be fitted on the "train"
- [ ] cluster "learning curves sets" and identify the properties of each cluster (eg, crossings, all good, all bad, god vs bad)
 
**Paper TODOs**
- [ ] review abstract + introduction @fmohr 
- [ ] review related work section @fmohr  + @tomviering  (for respective work on 2 surveys)
- [ ] review method section for successive halving, learning curve extrapolation and constant fidelity @tomviering 
- [ ] write the formula of the scores based on regret @tomviering 
- [ ] for each benchmark describe how the validation and test splits/performances are created
